src/support/examples/triangle/shader/triangle_110.glslv

```diff
@@ -1,8 +1,8 @@
-#version 150 core
+#version 110
```

Comment:
we still need GLSL 150 versions of triangle shaders, since OSX doesn't support compatibility GL profile and can't run GLSL 110 shaders...